sources/packages/api/src/route-controllers/education-program/models/save-education-program.dto.ts

```diff
@@ -119,22 +122,22 @@ export const transformToEducationProgramData = (
     credentialTypeToDisplay: credentialTypeToDisplay(program.credentialType),
     institutionId: program.institution.id,
     institutionName: program.institution.legalOperatingName,
-    submittedOn: program.submittedOn,
+    submittedOn: program.submittedDate,
     submittedBy: getUserFullName(program.submittedBy),
     effectiveEndDate: getISODateOnlyString(program.effectiveEndDate),
-    statusUpdatedOn: program.statusUpdatedOn,
+    assessedDate: program.assessedDate,
     // TODO: for now - program.effectiveEndDate is added by the ministry user
     // so, if program.effectiveEndDate is null/undefined, then
     // the program was auto approved, when institution submitted the
     // program, else the program was approved by ministry user.
     // ministry user uses IDIR. Program will always denied by
     // ministry user (i.e IDIR). Will need to update in future as
     // proper decision is taken
-    statusUpdatedBy:
+    assessedBy:
       program.effectiveEndDate ||
-      program.approvalStatus === ApprovalStatus.denied
-        ? getIDIRUserFullName(program.statusUpdatedBy)
-        : getUserFullName(program.statusUpdatedBy),
+      program.programStatus === ProgramStatus.Declined
```

Comment:
Can we please get rid of this logic and just return the `getUserFullName(educationProgram.assessedBy`?